pinecone/config/pinecone_config.py

```diff
@@ -0,0 +1,10 @@
+import os
+from .config import Config
+
+DEFAULT_CONTROLLER_HOST = "https://api.pinecone.io"
+
+
+class PineconeConfig(Config):
+    def __init__(self, api_key: str = None, host: str = None, **kwargs):
+        host = host or kwargs.get("host") or os.getenv("PINECONE_CONTROLLER_HOST") or DEFAULT_CONTROLLER_HOST
```

Comment:
I don't know if we want to keep this or not, but in the past there was an env var called `PINECONE_CONTROLLER_HOST` being used.

But as you can see, the only difference between a `Config` instance and the `PineconeConfig` subclass is a little bit of logic here about how to set the host value.